components/commons/modals/CloseButton.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+function CloseButton({ closeModal }: { closeModal: () => void }) {
```

Comment:
CloseButton은 Close를 암시하는 모양으로 생겼을 지라도, 단순한 버튼입니다. (모달과 무관하게 써도 되고, 그래도 될거같이 생겼고, 그렇게 재사용할 가치가 충분해보이는 구현입니다.)
지금은 클릭했을 때 무엇을 할지는 따로 정의되어있지 않으니, prop으로 오는 값에따라 달라집니다.

prop으로 뭘할지는 외부에서 결정하고 있으니까 (어떤 함수를 전달할지는 내부에서 알수도없고 강제할수도 없으니까) 이름은 명확하게 뭔지만 알려주는게 좋겠습니다. "니가 뭘할 지는 모르겠지만 난 버튼이니까  클릭하면 호출해줄게"

```suggestion
function CloseButton({ onClick }: { onClick: () => void }) {
```